examples/apps/color_finder.py

```diff
@@ -673,9 +674,15 @@ def get_blob_center(self):
         return blob_center
 
     def update_largest_blob_size(self, key):
+        '''Updates self.largest_blob_size based on the current largest blob.'''
         self.largest_blob_size = len(self.blobs_dict[key])
 
     def get_blob_size(self):
+        '''Gets the number of pixels in the largest blob.
+
+        Returns:
+            int specifying the size of the blob
```

Comment:
docstring for return value should be: "type: description" - e.g:
`int: The size, in pixels, of the largest blob.`